### DISCUSSION TITLE
Access subschema from error context?

### DISCUSSION THREAD

**Discussion by adjenks** [justinrainbow/json-schema#760] [Ideas]
Schemas can have title properties, as well as description. When an error is returned, I would like to get access to the title and possibly also the description field of that subschema where the error was identified. Is this possible? Currently we get the property, pointer, message and constraint. However, in a nested schema, the pointer doesn't really tell you where to look within the schema itself, so it's difficult to recover the descriptive properties, or extended ("x-property") properties, of a schema. Is there a way to access these in the context of an error that I'm just not noticing?

**Comment by erayd** [comment]
Can you give an example schema where the pointer is inadequate?

**Comment by adjenks** [comment]
Every schema. The pointer is a pointer for the data structure, not the schema itself.
I want the properties from [9.1](https://json-schema.org/draft/2019-09/json-schema-validation.html#rfc.section.9.1) and possibly other properties from the subschema that contains an error. So, when you get an error, how do you inspect the schema, not the data, at the point the error was identified?

**Comment by erayd** [comment]
Gotcha - I don't think we currently have a mechanism for that. You're welcome to submit a PR that implements it though.

Please note that if you do this, performance is a consideration.

**Comment by DannyvdSluijs** [comment]
@adjenks would this still be something you would consider following up on?

At the moment I'm very happy we are able to cleanup the repo and have revived it. 
This still didn't solve the problem with the limited time available from the maintainers. So as long 
as that remains a problem I'm afraid there is no one besides you that would pick-up this feature.

**Comment by adjenks** [comment]
@DannyvdSluijs I don't think I have the time right now either unfortunately.
I think the ticket should just be left open because I think it would still be a good feature but for now we can just hope someone will pick it up or I will get time eventually.

**Comment by DannyvdSluijs** [comment]
@adjenks I’ve migrated this one to a discussion to keep the issue list clear for long term ideas that might of might not be picked up
